core/shared/src/main/scala/scalaz/zio/duration/Duration.scala

```diff
@@ -19,6 +20,12 @@ sealed trait Duration extends Ordered[Duration] with Serializable with Product {
     case f: Duration.Finite => finite(f)
   }
 
+  final def toScalaDuration: ScalaDuration =
```

Comment:
asScala?

Reply:
In that case we might want to change `fromScalaDuration` to `fromScala`.